snowpack/assets/snowpack-init-file.js

```diff
@@ -0,0 +1,11 @@
+// Snowpack Configuration File
```

Comment:
Open to any and all bikeshedding on this file format!

Reply:
This makes sense to me